src/main/java/io/github/hildi/can/service/JavaSerializationService.java

```diff
@@ -0,0 +1,239 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.InvalidFileException;
+import io.github.hildi.can.exceptions.NotReadableFileException;
+import io.github.hildi.can.exceptions.NotWritableFileException;
+import io.github.hildi.can.exceptions.SerializationException;
+import io.github.hildi.can.model.FullName;
+import io.github.hildi.can.model.User;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by Serhii Hildi on 26.03.19.
+ */
+public class JavaSerializationService implements SerializationService{
+
+    private User user;
```

Comment:
It's not a good idea to have a service class with the state for the number of reasons:

First of all, you will not be able to use this service simultaneously from different threads.

Second, even in the single-threaded environment, if you want to avoid side effects - you will need to properly cleanup state (fields values) between invocations.

The best practice here - use local method variables instead of fields, wherever possible.